### PR TITLE
Fix: disklessset - Reintegrate PR

### PR DESCRIPTION
**Pull request by oxedions**
Re-integrating some PR that were lost.

### CODE REVIEW

File: disklessset/diskless/kernel_manager.py [hunk 1]
```diff
@@ -78,6 +78,8 @@ def change_kernel(image, kernel):
 
         # Use image method to set kernel
         image.kernel = kernel
+        # Update initramfs for selected kernel
```
Comment (ginomcevoy) [inline disklessset/diskless/kernel_manager.py:81]: Hello @oxedions 

The original PR was like this:
```
        # Update initramfs for selected kernel
        image.image = 'initramfs-kernel-' + kernel.replace('vmlinuz-', '')
        # Regenarate ipxe boot for the image
        image.generate_ipxe_boot_file()
        # Register image with new kernel
        image.register_image()
```
The order of the statements has changed (I am not sure if this order matters).

**Review by ginomcevoy — APPROVED** [review]
Hello @oxedions ,
Thank you for this reintegration! There is only a detail in kernel_manager.py about two lines changing order, but I inspected the code and it looks completely harmless.

File: disklessset/diskless/kernel_manager.py (hunk 1 above)
Comment (oxedions) [reply disklessset/diskless/kernel_manager.py:81]: Hello @ginomcevoy 
You are right, I didn't know exactly all the implication. I am waiting for @pietersdavid to confirm this before merging.
@pietersdavid ?

Comment (ginomcevoy) [reply disklessset/diskless/kernel_manager.py:81]: Looks good to me, hopefully we can merge soon.

Comment (oxedions) [reply disklessset/diskless/kernel_manager.py:81]: Yes, I will try to do it today. Need to resolve conflict by command line.

**Pull request merged**